test/agrovoc-allegrograph_on_hold.py

```diff
@@ -539,15 +523,15 @@ def testConstructByGETinJSON(self):
         ct = result.info()["content-type"]
         assert True in [one in ct for one in _SPARQL_DESCRIBE_CONSTRUCT_POSSIBLE], ct
         results = result.convert()
-        self.assertEqual(type(results), ConjunctiveGraph)
+        self.assertEqual(type(results), bytes)
```

Comment:
Not entirely clear why this result should be bytes?

Reply:
Because response is returned as not ConjunctiveGraph but bytes.

```
$ python -m unittest test.agrovoc-allegrograph_on_hold.SPARQLWrapperTests.testConstructByGETinJSON
/Users/eggplants/prog/sparqlwrapper/SPARQLWrapper/Wrapper.py:779: RuntimeWarning: Sending Accept header '*/*' because unexpected returned format 'json' in a 'CONSTRUCT' SPARQL query form
  warnings.warn(
/Users/eggplants/prog/sparqlwrapper/SPARQLWrapper/Wrapper.py:1162: RuntimeWarning: Format requested was JSON, but N3 (text/turtle; charset=utf-8) has been returned by the endpoint
  warnings.warn(
.
----------------------------------------------------------------------
Ran 1 test in 0.928s

OK
```

It seems that when the test was created, ConjunctiveGraph was returned, but now bytes are returned. Should I change the library so that ConjunctiveGraph is returned?
（The same reasoning applies to your reviews.）

Reply:
it is more a question of why it changed from one to the other, maybe it is better to leave it as it was and mark it with [@unittest.expectedFailure](https://docs.python.org/3/library/unittest.html#unittest.expectedFailure) until we are clear on why the change in type occurred.

Reply:
8347df7659fa64bd98bf892426e6223942dc0856